apps/root/src/common/components/news-banner/index.tsx

```diff
@@ -1,9 +1,9 @@
-// import { NETWORKS } from '@constants';
-// import { DCA_CREATE_ROUTE } from '@constants/routes';
-// import usePushToHistory from '@hooks/usePushToHistory';
+import { NETWORKS } from '@constants';
```

Comment:
Should we have different components for each banner and just change which one renders? so we dont have to keep commenting/uncommenting stuff on this file

Reply:
I think we could accumulate a lot of files doing so creating one component for each campaing. But I managed to abstract the parameters and it's more usable now for DX